lib/icingadb/redisconnection.cpp

```diff
@@ -729,3 +783,15 @@ void RedisConnection::RecordAffected(RedisConnection::QueryAffects affected, dou
 		}
 	}
 }
+
+/**
+ * Assert that the connection is not stopped.
+ *
+ * @throws RedisDisconnected if the connection is stopped.
+ */
+void RedisConnection::AssertNotStopped() const
+{
+	if (m_Stopped) {
+		throw RedisDisconnected();
+	}
+}
```

Comment:
`ThrowIfStopped()` might be a better name to avoid confusing it with actual asserts that aren't active in release builds and would otherwise terminate the process.

Reply:
Done.